dashboard/utils.py

```diff
@@ -123,12 +123,12 @@ def get_all_modules(module_type):
             logger.info(
                 f"Module {name} cannot be loaded because "
                 "we cannot import all dependencies. Download "
```

Comment:
Can we say. "Install this module using `pip....` instead of download?

Reply:
Done!